src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -129,24 +129,10 @@ suite('Kernel Execution @kernelCore', function () {
         await kernelExecution.executeCell(cell);
 
         assert.isAtLeast(cell.executionSummary?.executionOrder || 0, 1);
-        assert.strictEqual(
-            cell.outputs.length,
-            1,
-            `Cell should have one output, but has ${cell.outputs.length}, ${cell.outputs
-                .map((item) => `Cell Output items => ${item.items.map((oi) => oi.mime).join('')}`)
-                .join(', ')}`
-        );
-        assert.strictEqual(
```

Comment:
no point testing number of outputs as the code to merge outputs is in vscode and these tests don't rely on vscode controllers